content/karpenter/050_karpenter/ec2_spot_deployments.md

```diff
@@ -5,33 +5,27 @@ weight: 70
 
-Amazon EC2 terminates, your Spot Instance when Amazon EC2 needs the capacity back (or the Spot price exceeds the maximum price for your request). More recently Spot instances support also instance rebalancing recommendation. Amazon EC2 emits an instance rebalance recommendation signal to notify you that a Spot Instance is at an elevated risk of interruption. This signal gives you the opportunity to proactively rebalance your workloads across existing or new Spot Instances without having to wait for the two-minute Spot Instance interruption notice.
+Amazon EC2 terminates, your Spot Instance when Amazon EC2 needs the capacity back (or the Spot price exceeds the maximum price for your request). More recently Spot instances also support instance rebalance recommendations. Amazon EC2 emits an instance rebalance recommendation signal to notify you that a Spot Instance is at an elevated risk of interruption. This signal gives you the opportunity to proactively rebalance your workloads across existing or new Spot Instances without having to wait for the two-minute Spot Instance interruption notice.
 
-To capture this signals and handle graceful termination of our nodes, we will deploy a project called **[AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler)**. Node termination handler operates in two different modes Queue Mode and Instance Metadata Mode. We will use Instance Metadata mode. In this mode the aws-node-termination-handler [Instance Metadata Service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) Monitor will run a small pod ([DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)) on each host to perform monitoring of IMDS paths like /spot or /events and react accordingly to drain and/or cordon the corresponding node.
+## Karpenter and Spot Interruption
 
-To deploy the Node Termination Handler run the following command:
-```
-helm repo add eks https://aws.github.io/eks-charts
-helm install aws-node-termination-handler \
-             --namespace kube-system \
-             --version 0.19.2 \
-             --set nodeSelector."karpenter\\.sh/capacity-type"=spot \
-             eks/aws-node-termination-handler
-```
+Karpenter natively handles Spot Interruption Notifications by consuming events from an SQS queue which is populated with Spot Interruption Notifications via EventBridge. All of the infrastructure is setup by Karpenter's CloudFormation template that was applied previously. When Karpenter receives a Spot Interruption Notification, it will gracefully drain the interrupted node of any running pods while also provisioning a new node for those pods to quickly schedule onto.
 
-{{% notice note %}}
-The helm command above does make use of the `nodeSelector` pointing to `kubernetes.sh/capacity-type: spot`. This way will only install the node-termination-handler in Spot nodes.
-{{% /notice %}}
+## Deploy Node-Termination-Handler on EC2 Spot Instances
```

Comment:
I'd suggest modifying the title... As we are not deploying anymore. We can still talk about Node termination handler and how it is integrated to take care of terminations.

Reply:
Done!